webapp/components/wpt-results.js

```diff
@@ -204,11 +204,21 @@ class WPTResults extends WPTColors(WPTFlags(SelfNavigation(LoadingState(TestRuns
         <div class="links">
           <ul>
             <li><a href\$="https://github.com/web-platform-tests/wpt/blob/master[[sourcePath]]" target="_blank">View source on GitHub</a></li>
-            <template is="dom-if" if="{{ testW3CURL }}">
-              <li><a href="[[testW3CURL]]" target="_blank">Run in your browser on w3c-test.org</a></li>
+            <template is="dom-if" if="{{ showTestURL }}">
+              <template is="dom-if" if="{{ !webPlatformTestsLive }}">
```

Comment:
You shouldn't need to duplicate the `dom-if` here; you've already forked what the URL computes to (so just revert lines 207 to 212.

Reply:
I thought I have to dom-if because
1) showTestURL could return empty string as line 530
2) the text when the flag is on/off is different - Run in your browser on w3c-test.org vs Run in your browser on web-platform-tests.live

Reply:
Oh, the anchor text. In that case, extract the host out of the anchor's sentence into a variable, i.e.

    Run in your browser on [[ liveTestDomain ]]

(and add a computed property for liveTestDomain).

And then we don't need the dom-if's.